### DISCUSSION TITLE
Can't run new added certa tests

### DISCUSSION THREAD

**Discussion by dbiguenet** [iTwin/itwinjs-core#4592] [Q&A] [answered]
Hi everyone.
I’m working on a new API sdk, everything is working, so the next step is to add some integration tests, using certa.
I added a certa.json file, a webpack.config.js file in my node project, however, when I run “certa -r node” command, I get this error : 
`cleanUpCallback is not a function
TypeError: cleanUpCallback is not a function
    at certa (C:\Users\Denis.Biguenet\Documents\Acute3D\reality-apis-samples\typescript\SDK\node_modules\@itwin\certa\src\CertaCore.ts:63:11)    
    at C:\Users\Denis.Biguenet\Documents\Acute3D\reality-apis-samples\typescript\SDK\node_modules\@itwin\certa\bin\certa.js:93:5`

Here is my code https://github.com/iTwin/reality-apis-samples/tree/dbiguenet/Tests/typescript/SDK/src/Tests 
Can you help me?

**Comment by aruniverse** [comment, marked as answer]
Thanks for reporting this. Will be fixed in 3.5.0 and a patch in 3.4.5 with these [changes](https://github.com/iTwin/itwinjs-core/pull/4636)
